ci/deploy.sh

```diff
@@ -4,27 +4,8 @@ set -eu -o pipefail # Exit with nonzero exit code if anything fails
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 
-is-automatic-update() {
-  [[ "$TRAVIS_EVENT_TYPE" == "cron" ]] || [[ "$TRAVIS_EVENT_TYPE" == "api" ]]
-}
-
-if is-automatic-update; then
-  keys_dir=$(mktemp -d)
-  openssl aes-256-cbc \
-    -K $encrypted_080f214a372c_key \
-    -iv $encrypted_080f214a372c_iv \
-    -in ci/keys.tar.gz.enc -out ci/keys.tar.gz -d
-  tar -C "$keys_dir" -xzvf ci/keys.tar.gz
-
-  eval "$(ssh-agent -s)"
-
-  chmod 600 "${keys_dir}/ssh-key"
-  ssh-add "${keys_dir}/ssh-key"
-  gpg --import "${keys_dir}/gpg-private-key"
-  rm -rf "$keys_dir"
-fi
-
-export encrypted_080f214a372c_key= encrypted_080f214a372c_iv=
+source ${DIR}/lib/travis-functions.sh
+source ${DIR}/lib/setup-git.sh
```

Comment:
I think this line can be also moved after dont-continue-on-pull-requests. I have not checked

Reply:
yeah, but the code shows that it is needed to pull `nur-combined`

Reply:
Yes, we never pull nur-combined on PRs though.